configure.ac

```diff
@@ -61,6 +63,7 @@ AS_IF([test -n "$missing_gmock"],
 # Checks for libraries.
 AC_SEARCH_LIBS([__b64_ntop], [resolv])
 AC_SEARCH_LIBS([pthread_create], [pthread])
+AC_SEARCH_LIBS([evhtp_ssl_init], [evhtp])
```

Comment:
That's a public API function, but it's used for writing SSL *servers* instead.
Changed to `evhtp_make_request`.